safe/definitions/exposure.py

```diff
@@ -47,22 +49,43 @@
     ],
     'earthquake_notes': [
         # these are earthquake specific notes for population
-        tr('Map shows the estimation of displaced population.'),
-        tr('People are displaced if they experience and survive a shake level '
-           'of more than 5 on the MMI scale.'),
+        tr('Map shows the estimated displaced population.People are displaced '
+           'if they experience and survive a shake level of more than 5 on '
+           'the MMI scale.'),
+        tr('Exposed population varies by the time (day or night, weekends, '
+           'holidays etc.). Such variations are not considered in the '
+           'estimates in the InaSAFE.'),
         tr('The fatality calculation assumes that no fatalities occur for '
            'shake levels below 4 and fatality counts of less than 50 are '
-           'disregarded.'),
+           'rounded down.'),
 
     ],
     'earthquake_pager_notes': [   # these are earthquake Pager specific notes
         tr('Fatality model is from Population Vulnerability '
             'Pager Model.'),
     ],
     'earthquake_itb_notes': [   # these are earthquake ITB specific notes
-         tr('Fatality model is from Institut Teknologi Bandung 2012.'),
+        tr('Fatality model is from Institut Teknologi Bandung 2012.'),
 
     ],
+    'earthquake_fatality_model_limitations': [# notes provided by Hadi Ghasemi
```

Comment:
Travis said : 
> ./safe/definitions/exposure.py:71:47: E261 at least two spaces before inline comment

So you need : 
`'earthquake_fatality_model_limitations': [  # notes provided by Hadi Ghasemi`